go.mod

```diff
@@ -174,6 +174,8 @@ require (
 	github.com/mailgun/minheap v0.0.0-20170619185613-3dbe6c6bf55f // indirect
 )
 
+require github.com/golang/protobuf v1.5.3 // indirect
```

Comment:
Do you mind reverting this change? It's best to keep it in the big indirect block below, with the others.